packages/zosfiles/src/index.ts

```diff
@@ -42,8 +42,4 @@ export * from "./methods/upload";
 export * from "./methods/utilities";
 
 export * from "./utils/ZosFilesAttributes";
-export * from "./utils/ZosFilesUtils";
-export * from "./doc/IOptionsFullResponse";
-export * from "./doc/IRestClientResponse";
-export * from "./doc/IOptions";
-export * from "./doc/types/ZosmfRestClientProperties";
```

Comment:
Curious why these were removed instead of commented out like the rest? 😅 
Should we remove instead of comment-out?

Reply:
Those were all duplicates. They all appear earlier in the file, and were exported twice.